src/metabase/api/card.clj

```diff
@@ -468,4 +469,5 @@
   (db/select [Card :name :id], :enable_embedding true, :archived false))
 
 
-(api/define-routes)
+(api/define-routes
+  (middleware/streaming-json-response POST_:card-id_query))
```

Comment:
That sounds much safer, it's unhygenic to leak auto-genrated symbols. I'll do that